index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Sai Samant</title>
+  </head>
+  <body>
+
+    <header>
+      <div class="hello">
+        <h1>&lt; hello. &gt;</h1>
+      </div>
+      <div class="nav">
```

Comment:
Why have a class called `.nav`? Could you avoid the `div` and style the more semantic `<nav>` tag directly?